modules/component/spec/let/let.directive.spec.ts

```diff
@@ -399,6 +432,28 @@ describe('LetDirective', () => {
     });
   });
 
+  describe('when a standalone component is used', () => {
```

Comment:
I don't think we need to test whether the standalone component can be used with the standalone directive. This is already tested in the Angular repo, so we can remove these tests.